src/app/posts/components/post-item/post-item.component.html

```diff
@@ -1,28 +1,23 @@
-<article class="post">
-  <div class="post__body">
-    <div class="post__head">
-      <div class="post__author-block">
-
-        <div class="post__author-img">
-          <img src="../../assets/no-avatar.svg" [alt]="post.author" [title]="post.author">
-        </div>
-
-        <div class="post__author-info">
-          <div class="post__author-name">{{post.author}}</div>
-          <div class="post__author-date">{{post.date | date:"MMMM d, YYYY"}}</div>
-        </div>
-      </div>
-
-      <div class="post__like-block">
-        <div class="post__like post__like_active">&#9829; {{post.likeNumber}}</div>
+<article>
```

Comment:
check if`` post``exists here. After adding HTTPClient to the project you will have an error during time, when you wait a response from backend